Error setting title for chat view

Via GPlay console:
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3431)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3595)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:85)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2066)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:223)
  at android.app.ActivityThread.main (ActivityThread.java:7664)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:592)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:947)
Caused by: java.lang.IllegalStateException: 
  at androidx.core.util.Preconditions.checkState (Preconditions.java:151)
  at androidx.emoji.text.EmojiCompat.process (EmojiCompat.java:742)
  at androidx.emoji.text.EmojiCompat.process (EmojiCompat.java:702)
  at androidx.emoji.text.EmojiCompat.process (EmojiCompat.java:666)
  at androidx.emoji.text.EmojiCompat.process (EmojiCompat.java:634)
  at com.nextcloud.talk.controllers.ChatController.getTitle (ChatController.kt:1273)
  at com.nextcloud.talk.controllers.base.NewBaseController.isTitleSetable (NewBaseController.kt:251)
  at com.nextcloud.talk.controllers.base.NewBaseController.setTitle (NewBaseController.kt:228)
  at com.nextcloud.talk.controllers.ChatController.onViewBound (ChatController.kt:738)
  at com.nextcloud.talk.controllers.base.NewBaseController$1.postCreateView (NewBaseController.kt:98)
  at com.bluelinelabs.conductor.Controller.inflate (Controller.java:1069)
  at com.bluelinelabs.conductor.ControllerChangeHandler.executeChange (ControllerChangeHandler.java:195)
  at com.bluelinelabs.conductor.ControllerChangeHandler.executeChange (ControllerChangeHandler.java:159)
  at com.bluelinelabs.conductor.Router.performControllerChange (Router.java:847)
  at com.bluelinelabs.conductor.Router.performControllerChange (Router.java:807)
  at com.bluelinelabs.conductor.Router.rebindIfNeeded (Router.java:553)
  at com.bluelinelabs.conductor.Conductor.attachRouter (Conductor.kt:30)
  at com.nextcloud.talk.activities.MainActivity.onCreate (MainActivity.kt:101)
  at android.app.Activity.performCreate (Activity.java:7994)
  at android.app.Activity.performCreate (Activity.java:7978)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1309)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3404)
```

--- a/app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt
+++ b/app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt
@@ -1270,7 +1270,11 @@ class ChatController(args: Bundle) :
     override val title: String
         get() =
             if (currentConversation?.displayName != null) {
-                " " + EmojiCompat.get().process(currentConversation?.displayName as CharSequence).toString()
+                try {
+                    " " + EmojiCompat.get().process(currentConversation?.displayName as CharSequence).toString()
+                } catch (e: IllegalStateException) {
+                    " " + currentConversation?.displayName
+                }
             } else {
                 ""
             }

